pkg/trace/stats/client_stats_aggregator.go

```diff
@@ -0,0 +1,283 @@
+package stats
+
+import (
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/trace/config"
+	"github.com/DataDog/datadog-agent/pkg/trace/info"
+	"github.com/DataDog/datadog-agent/pkg/trace/pb"
+	"github.com/DataDog/datadog-agent/pkg/trace/watchdog"
+)
+
+const (
+	aggBucketSize           = uint64(1e9)
+	bucketDuration          = uint64(1e10) // 10s
+	oldestBucketStartSecond = uint64(20)   // 20s
+	distributionsPayload    = "distributions"
+	aggregatedCountPayload  = "counts"
```

Comment:
```suggestion
	keyDistributions    = "distributions"
	keyCounts  = "counts"
```
Let's put these in a different `const ( ... )` group and document all of these constants.